mappings/net/minecraft/block/Blocks.mapping

```diff
@@ -1,4 +1,26 @@
 CLASS net/minecraft/class_2246 net/minecraft/block/Blocks
+	METHOD method_26106 getLeavesSettings ()Lnet/minecraft/class_2397;
+	METHOD method_26107 getLightLevelFromBlockState (I)Ljava/util/function/ToIntFunction;
+		ARG 0 level
```

Comment:
Maybe 
```suggestion
		ARG 0 litLevel
```
as the result function only returns this level when the block state has "lit" property's value equal to "true"